coherent state constellations

```diff
@@ -0,0 +1,42 @@
+code_id: coherent state constellations
+physical: qubits
+logical: bits
+
+# Apostrophes are denoted by two apostrophe characters, i.e., ''
+# Code title (SINGULAR) + first reference(s) (optional).
+name: 'coherent state constellations for Bosonic Gaussian channels'
+introduced: '\cite{doi:10.1109/ISIT.2016.7541749}'
+
+# Anything applicable to a larger parent set of codes (see below) should go in
+# that entry instead of here.
+description: |
+    The capacity of quantum channels has not been stablished as good as classical channels, but for the case of coherent information restricitng to Gaussian inputs, the maximum is achieved when the input is in the form of thermal state. Thermal state can be created by adding infinite coherent state which in practice, is not attainable since both our systems and coding methods are finite-dimensional. This method developes a discrerization method to use a limited number of coherent state, a constellation, to replicate the thermal state to some extent. Having a finite-dimensional quantum setup, existing techniques for constructing well-performing codes can be applied. For example we can construct quantum polar codes that achieves the coherent information of thermal noise channel bound. 
```

Comment:
This is a pretty good intro, but it's touching on only the thermal state aspect. Please comment on how such thermal states can be used to construct codes.